hadoop-ozone/client/src/main/java/org/apache/hadoop/ozone/client/OzoneKey.java

```diff
@@ -74,11 +74,18 @@ public class OzoneKey {
    * Constructs OzoneKey from OmKeyInfo.
    *
    */
```

Comment:
This comment belongs to the constructor below the new member var.

Reply:
I think this belongs to the static method at the bottom of the class definition:

```
 public static OzoneKey fromKeyInfo(OmKeyInfo keyInfo)
```

I have moved it there.

Reply:
Sorry, I didn't notice it was misplaced even before the patch. :)

Reply:
I didn't even check that - but I did wonder how I managed to move it from one end of the file to the other.